tests/pytests/unit/modules/test_schedule.py

```diff
@@ -93,39 +95,41 @@ def test_delete(sock_dir, job1):
     """
     Test if it delete a job from the minion's schedule.
     """
-    with patch.dict(schedule.__opts__, {"schedule": {}, "sock_dir": sock_dir}):
-        mock = MagicMock(return_value=True)
-        with patch.dict(schedule.__salt__, {"event.fire": mock}):
-            _ret_value = {"complete": True, "schedule": {}}
-            with patch.object(SaltEvent, "get_event", return_value=_ret_value):
-                assert schedule.delete("job1") == {
-                    "comment": "Job job1 does not exist.",
-                    "changes": {},
-                    "result": False,
-                }
+    with patch("os.makedirs", MagicMock(return_value=True)):
+        with patch.dict(schedule.__opts__, {"schedule": {}, "sock_dir": sock_dir}):
+            mock = MagicMock(return_value=True)
+            with patch.dict(schedule.__salt__, {"event.fire": mock}):
+                _ret_value = {"complete": True, "schedule": {}}
+                with patch.object(SaltEvent, "get_event", return_value=_ret_value):
```

Comment:
**suggestion (non-blocking)** another opportunity for patching. Though, seeing these several patches, would it be possible to extract these into a test fixture, or would that bork up other parts of these existing test functions?